tests/models/test_hooks.py

```diff
@@ -483,9 +482,6 @@ def teardown(self, stage: str):
         'on_epoch_end',
         'on_test_end',
         'on_test_model_train',
-        'on_fit_end',
-        'teardown',  # for 'fit'
-        'teardown',  # for 'test'
+        'teardown',
     ]
-
-    assert model2.called == expected
+    assert model.called == expected
```

Comment:
reminder: we should rewrite this test with a model mock for clarity and sanity :))

Reply:
want me to work on it?

Reply:
sure!